### PR TITLE
SDI-284 Allow booking overrides for offender court cases

### PR DESCRIPTION
**Pull request by mikehalmamoj**

### CODE REVIEW

**Review by petergphillips — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java [hunk 1]
```diff
@@ -705,7 +705,7 @@ public MilitaryRecords getMilitaryRecords(final Long bookingId) {
                 )).orElseThrow(EntityNotFoundException.withMessage("Offender booking with id %d not found.", bookingId));
     }
 
-    @VerifyBookingAccess
+    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
```
Comment (andymarke) [inline src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java:708]: maybe drop GLOBAL_SEARCH given this is not related to search at all
